.github/workflows/python-app.yml

```diff
@@ -23,8 +23,6 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python3 -m venv ~/gmxenv
```

Comment:
While testing `gmx2qmmm` locally, I usually install it in a separate Python environment. Would you consider it as a recommended installation procedure?

1. If so, wouldn't it make sense to test it on GitHub runners in a separate Python environment as well (though it didn't work for some reason)?

2. Installing everything now in a separate environment, I noticed that `pytest` was not installed alongside. Shouldn't it be marked as a required dependency in `setup.cfg`? Do we also need a `requirements.txt` file or it is an optional practice?

Reply:
So if you work with gmx2qmmm locally, I would always recommend using a virtual environment to keep on top of your dependencies. This is especially the case for development but also in production it makes sense to have dedicated environments for different use-cases.

In the CI, it currently does not matter because the CI environment is just used once. You can use an environment but there is no advantage.

A reason why the env did not work for you before might be that you where mixing `python `and `python3 `commands.

If we want to test against multiple versions of Python at some point, we could use a tool like nox that takes care of creating virtual environments as needed.

`pytest` is just an optional dependency for development. It is listed in `setup.cfg` under `"options.extras_require"`. You can install it on demand with `pip install .[test]`. In the CI, I only install it, once the test step is entered.